ledger/ledger-api-tests/infrastructure/src/main/scala/com/daml/ledger/api/testtool/infrastructure/participant/SingleParticipantTestContext.scala

```diff
@@ -97,52 +97,27 @@ import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Failure
 import scala.util.control.NonFatal
 
-private[testtool] object ParticipantTestContext {
-
-  private[this] def filter(templateIds: Seq[Identifier]): Filters =
-    new Filters(
-      if (templateIds.isEmpty) None
-      else Some(new InclusiveFilters(templateIds))
-    )
-
-  private def transactionFilter(
-      parties: Seq[String],
-      templateIds: Seq[Identifier],
-  ): Some[TransactionFilter] =
-    Some(new TransactionFilter(Map(parties.map(_ -> filter(templateIds)): _*)))
-
-}
-
 /** Exposes services running on some participant server in a test case.
   *
-  * Each time a test case is run it receives a fresh instance of [[ParticipantTestContext]]
+  * Each time a test case is run it receives a fresh instance of [[SingleParticipantTestContext]]
   * (one for every used participant server).
   */
-final class ParticipantTestContext private[participant] (
+final class SingleParticipantTestContext private[participant] (
     val ledgerId: String,
     val endpointId: String,
-    val applicationId: String,
+    override val applicationId: String,
```

Comment:
Can you use `override` for all or for none (if it is not a true implementation override)? It looks.. non-regular and on closer inspection I can't find a good reason for it :D

Reply:
Sure, would make sense. Added it as it;s something we usually add all the time in the integration kit